### PR TITLE
Fixing URL for AUTHENTICATION_API_HOST on test

### PR DESCRIPTION
**Pull request by srh-sloan**
Also fixes the assets build which was failing as FLASK_ENV wasn't set

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
       - name: install dependencies
         run: python -m pip install --upgrade pip && python -m pip install -r requirements-dev.txt
       - name: build static assets
-        run: python build.py
+        run: FLASK_ENV=development python build.py
```
Comment (gidsg) [inline .github/workflows/main.yml:28]: Is this a regression since the common config stuff was added? Can we default this to development?

Comment (srh-sloan) [reply .github/workflows/main.yml:28]: We stopped defaulting to development in common config and changed to throw a KeyError instead if it wasn't in the env, so we can't accidentally go into development on live. Consequence of that is doing this instead.

Comment (srh-sloan) [reply .github/workflows/main.yml:28]: We had the same issue with unit tests and got round this by setting FLASK_ENV in pytest.ini

**Review by gidsg — APPROVED** [review]
Fine to fix build! But as mentioned would be good to re-visit this common config change that is adding this complexity.

**Pull request merged**